posthog/cdp/templates/loops/test_template_loops.py

```diff
@@ -0,0 +1,61 @@
+from inline_snapshot import snapshot
+from posthog.cdp.templates.helpers import BaseHogFunctionTemplateTest
+from posthog.cdp.templates.loops.template_loops import template as template_loops
+
+
+class TestTemplateLoops(BaseHogFunctionTemplateTest):
+    template = template_loops
+
+    def _inputs(self, **kwargs):
+        inputs = {"apiKey": "1cac089e00a708680bdb1ed9f082d5bf"}
```

Comment:
Just checking - this isn't a real api key right?

Reply:
It was until I disabled that API key.